openapi/spec/openapi.json

```diff
@@ -47,6 +47,38 @@
         }
       }
     },
+    "/broadcasts/{broadcast_id}": {
+      "get": {
+        "x-beta": true,
+        "tags": ["Notification Broadcasts APIs"],
+        "operationId": "broadcasts-get",
+        "summary": "Fetch a notification broadcast by its ID",
+        "description": "Fetch a notification broadcast by its ID.",
+        "parameters": [
+          { "$ref": "#/components/parameters/x_magicbell_api_key_header" },
+          { "$ref": "#/components/parameters/x_magicbell_api_secret_header" },
+          { "$ref": "#/components/parameters/broadcast_id" }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "required": ["broadcasts"],
+                  "properties": {
+                    "broadcast": {
+                      "$ref": "#/components/schemas/Broadcast"
```

Comment:
The only difference between the schema for the LIST and FETCH endpoints for broadcasts is that I truncate the content for LIST to 160 char, to keep the API performant.